source/conditionals.rst

```diff
@@ -147,14 +154,29 @@ Europe  UK
 42      8
 ======  =====
 
+Extension: Can you return an error message for the user if they enter an invalid shoe size.
+
+Practical: Loan
```

Comment:
I wonder if we need a 'tip' section here to help nudge towards writing out the problem on paper (ie: work out the maths before trying to write it in code). That may just be my personal way of programming though...